lib/settings-observer.js

```
@@ -64,7 +64,7 @@ class SettingsObserver {
         this._handler.call(this, newVal, oldVal);
       }
     } catch (e) {
-      //
+      // Handle error to make sure handler do not stop others
```

We should just throw the exception if there's no handling?

---

no i do not think so => because it will stop next observer. I think we can send error to hander. e.g: handler(error, newVal, oldVal) => but with this approach we have to change our Crypto Miner code (change handler)

---

Oh, right this is in observer, we should prob just log it somehow